definitions/divorce/json/CaseEventToFields.json

```diff
@@ -6580,9 +6580,9 @@
     "PageLabel": "Refusal order: Reject",
     "PageDisplayOrder": 3,
     "PageShowCondition": "RefusalDecision=\"reject\"",
+    "ShowSummaryChangeOption": "Yes",
     "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/dn-about-to-be-granted",
-    "RetriesTimeoutURLMidEvent": "120,120",
-    "ShowSummaryChangeOption": "Yes"
+    "RetriesTimeoutURLMidEvent": "120,120"
```

Comment:
this happened automatically when generating the json, so not intentional

Reply:
thanks for letting know :) Am I correct - it is exactly the same, isn't it?

Reply:
Yes it's the same! Just a case of different ordering